solution/ui/regulations/eregs-vite/src/views/Subjects.vue

```diff
@@ -496,7 +525,10 @@ getDocSubjects();
                         </div>
                         <div class="subject__filters--row">
                             <DocumentTypeSelector v-if="isAuthenticated" />
-                            <FetchCategoriesContainer v-slot="slotProps">
+                            <FetchCategoriesContainer
+                                v-slot="slotProps"
+                                :categories-capture-function="setCategories"
```

Comment:
If a `content-search` response item is assigned to a subcategory, the included subcategory information (`id`, `name`, etc) no longer includes the parent category name.  It now only includes the parent category's id in the `parent` field.

Ex: `"parent": 4,`"

However, we need the name of this parent category to be displayed in the UI alongside the subcategory name.  As such, we need to fetch the categories ourselves from the API in order to use the `parent` id to find the parent's `name`.

We're already getting categories in this `FetchCategoriesContainer` component to be used in the categories dropdown menu.  We can use these fetched categories instead of making a second, separate API call; we just have to pass in a closure down to `FetchCategoriesContainer` that will update the parent scope when the child fetches the categories.

Admittedly, passing a function as a prop is considered an anti-pattern in Vue, but for this specific instance it can be tolerated.  We may also update the backend to return the parent category name instead of the id, but for now we must accept this implementation.